app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>My Recipes</h1>
+
+<ul>
+  <% @recipes.each do |recipe| %>
+    <li>
+      <%= link_to recipe.name, recipe %>
+      <%= button_to 'Delete', recipe, method: :delete, data: { confirm: 'Are you sure?' } if recipe.user == current_user %>
+    </li>
+  <% end %>
+</ul>
```

Comment:
- [ ] [REQUIRED] Great implementation so far team, however, we need to add the description and make sure we replicate the wireframe provided for the index page for recipes list as stated by the project requirement.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
<img width="599" alt="Screenshot 2023-09-16 at 22 49 29" src="https://github.com/techmoves/recipe-app/assets/58692590/70bacced-bb9a-4b70-a35f-8d834524270e"> | <img width="940" alt="Screenshot 2023-09-16 at 22 50 35" src="https://github.com/techmoves/recipe-app/assets/58692590/4d78751b-f9c7-4fef-970f-f457b44a9fb8">